### PR TITLE
feat: add verify by HTML File

### PR DESCRIPTION
**Pull request by datbnt-0905**

### CODE REVIEW

File: src/Models/DomainVerification.php [hunk 1]
```diff
@@ -3,8 +3,9 @@
 namespace SunAsterisk\DomainVerifier\Models;
 
 use Illuminate\Database\Eloquent\Model;
+use SunAsterisk\DomainVerifier\Contracts\Models\DomainVerifiableInterface;
 
-class DomainVerification extends Model
+class DomainVerification extends Model implements DomainVerifiableInterface
```
Comment (kimyvgy) [inline src/Models/DomainVerification.php]: It is the same issue in the pull #12. Please see [my comment](https://github.com/sun-asterisk-research/laravel-domain-verify/pull/12#discussion_r316944875).

File: src/Strategies/HTMLFile.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+use SunAsterisk\DomainVerifier\Contracts\Strategies\StrategyInterface;
+use SunAsterisk\DomainVerifier\DomainVerificationFacade;
+
+class HTMLFile implements StrategyInterface
+{
+    public function verify(string $url, DomainVerifiableInterface $domainVerifiable)
+    {
+        $verificationName = config("domain_verifier.verification_name");
+        $urlFile = $url.'./'.$verificationName.'.html';
+        $domainToken = @file_get_contents($urlFile);
```
Comment (kimyvgy) [inline src/Strategies/HTMLFile.php]: Should add a new method to get HTML content.

File: tests/Strategies/HTMLFileTest.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+class HTMLFileTest extends TestCase
+{
+    public function test_it_can_verify_html_file()
+    {
+        $url = 'https://domain.local';
+        $domain = m::mock(DomainVerification::class);
+        $html = m::mock(HTML::class);
+
+        $html->shouldReceive('verify')
```
Comment (kimyvgy) [inline tests/Strategies/HTMLFileTest.php]: Same issue #12

**Comment by datbnt-0905** [comment]
I fix as you say. Can you review it ?

**Pull request merged**